nvm.sh

```diff
@@ -3015,38 +3052,63 @@ nvm() {
       if [ -n "${NVM_LTS-}" ]; then
         VERSION="$(nvm_match_version "lts/${NVM_LTS:-*}")"
       elif [ -z "${PROVIDED_VERSION-}" ]; then
-        nvm_rc_version
+        # Get the .nvmrc version depending if quiet mode is on or not
+        if [ $NVM_USE_SILENT -ne 1 ]; then
+          nvm_rc_version
+        else
+          nvm_rc_version --silent
+        fi
         if [ -n "${NVM_RC_VERSION-}" ]; then
           PROVIDED_VERSION="${NVM_RC_VERSION}"
           VERSION="$(nvm_version "${PROVIDED_VERSION}")"
         fi
         unset NVM_RC_VERSION
         if [ -z "${VERSION}" ]; then
-          nvm_err 'Please see `nvm --help` or https://github.com/nvm-sh/nvm#nvmrc for more information.'
+          if [ $NVM_USE_SILENT -ne 1 ]; then
+            nvm_err 'Please see `nvm --help` or https://github.com/nvm-sh/nvm#nvmrc for more information.'
+          fi
           return 127
         fi
       else
         VERSION="$(nvm_match_version "${PROVIDED_VERSION}")"
       fi
 
       if [ -z "${VERSION}" ]; then
-        >&2 nvm --help
+        if [ $NVM_USE_SILENT -ne 1 ]; then
+          >&2 nvm --help
+        fi
```

Comment:
i don't think "--silent" should ever suppress help output.

Reply:
Ok, I noticed that the help went to standard error and many CLIs like NPM don't display stderr on silent mode and expect the users to find out by exit code on silent mode. But then that is a little bit in the grey area because errors are something everyone wants to take a look at. I will make a commit to revert that change.